docs/utilities/middleware_factory.md

```diff
@@ -19,6 +19,16 @@ Middleware factory provides a decorator factory to create your own middleware to
 
 You might need a custom middleware to abstract non-functional code. These are often custom authorization or any reusable logic you might need to run before/after a Lambda function invocation.
 
+### Environment variables
```

Comment:
What do you think of the "Environment Variables" section being the last of the Getting Started section? So we keep a standard.

Reply:
Thanks for addressing this feedback @barreeeiroo! :rocket: